subclass/thetoonwarlockpatron.json

```
@@ -0,0 +1,221 @@
+{
+    "_meta": {
+        "sources": [
+            {
+                "json": "MFOV:TT",
+                "abbreviation": "TTOP",
+                "full": "The Toon Otherworldly Patron",
+                "authors": [
+                    "Mage Hand Press",
+                    "Middle Finger of Vecna"
+                ],
+                "convertedBy": [
+                    "Sammun#0836"
+                ],
+                "version": "1.0.0",
+                "url": "https://mfov.magehandpress.com/2017/04/the-toon.html",
+                "targetSchema": "1.0.8"
+            }
+        ],
+        "dateAdded": 1549715846,
+        "dateLastModified": 1660453920
+    },
+    "subclass": [
+        {
+            "name": "The Toon",
+            "source": "MFOV:TT",
+            "className": "Warlock",
+            "classSource": "PHB",
+            "shortName": "The Toon",
+            "subclassSpells": [
+                "feather fall",
+                "tasha's hideous laughter",
+                "blur",
+                "rope trick",
+                "haste",
+                "stinking cloud",
+                "confusion",
+                "freedom of movement",
+                "mislead",
+                "passwall"
+            ],
+            "spells": [
+                {
+                    "expanded": {
+                        "1": [
+                            "feather fall",
+                            "tasha's hideous laughter"
+                        ],
+                        "2": [
+                            "blur",
+                            "rope trick"
+                        ],
+                        "3": [
+                            "haste",
+                            "stinking cloud"
+                        ],
+                        "4": [
+                            "confusion",
+                            "freedom of movement"
+                        ],
+                        "5": [
+                            "mislead",
+                            "passwall"
+                        ]
+                    }
+                }
+            ],
+            "subclassFeatures": [
+                "The Toon|Warlock||The Toon|MFOV:TT|1",
+                "Slaspstick|Warlock||The Toon|MFOV:TT|6",
+                "Two-Dimensional|Warlock||The Toon|MFOV:TT|10",
+                "Anvil on Head|Warlock||The Toon|MFOV:TT|14"
+
+            ]
+        }
+    ],
+    "subclassFeature": [
+        {
+            "name": "The Toon",
+            "source": "WarlockToon",

```

You need to change all the `WarlockToon` source references, as it does not exist. The only source in the meta is `MFOV:TT`:

```suggestion
            "source": "MFOV:TT",
```